test/Runtime/POSIX/Replay.c

```diff
@@ -1,10 +1,7 @@
 // RUN: %clang -DKLEE_EXECUTION %s -emit-llvm %O0opt -c -o %t.bc
 // RUN: rm -rf %t.klee-out
 // RUN: %klee --output-dir=%t.klee-out --posix-runtime %t.bc --sym-files 1 3
-// RUN: %klee-replay --create-files-only %t.klee-out/test000001.ktest
-
-// RUN: FileCheck --input-file=A --check-prefix=CREATE_FILES_ONLY %s
-// CREATE_FILES_ONLY: abc
+// RUN: %klee-replay --create-files-only %t.klee-out/test000001.ktest 2>%t.txt
```

Comment:
Good catch :)  Since now all files are created in a temporary directory, `--input-file=A` doesn't work anymore.  If we want to keep this check, we would either need to parse the output of `klee-replay` for the name of the temporary directory, or add a feature to `klee-replay` to let users specify the name of the directory.  But I'm not sure that feature would be useful outside this check.

Reply:
In that case I'm for removing this check. The remaining checks are sufficient. Please, just remove the `2>%t.txt` at the end.

Reply:
Done